wpilibj/src/main/java/edu/wpi/first/wpilibj/PWM.java

```diff
@@ -109,9 +104,9 @@ public void enableDeadbandElimination(boolean eliminateDeadband) {
    * @param deadbandMin The low end of the deadband pulse width in ms
    * @param min The minimum pulse width in ms
    */
-  public void setBounds(
-      double max, double deadbandMax, double center, double deadbandMin, double min) {
-    PWMJNI.setPWMConfig(m_handle, max, deadbandMax, center, deadbandMin, min);
+  public void setBoundsMicroseconds(
+      int max, int deadbandMax, int center, int deadbandMin, int min) {
+    PWMJNI.setPWMConfigMicroseconds(m_handle, max, deadbandMax, center, deadbandMin, min);
```

Comment:
The doc comment specifies the units as ms- Should this be changed to microseconds or μs?